<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>574</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>574</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>574</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="line_plot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="bar_plot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="pie_plot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="reg_plot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Line Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bar Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pie Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reg Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Histo Plot   </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Box Plot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heatmap Plot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="heatmap">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>220</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="box_plot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="histo_plot">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scatter Plot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="scatter_plot">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_plot">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&gt; Load &lt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>449</y>
      <width>731</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>731</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
